vars/rollingUpgradePipeline.groovy

```diff
@@ -60,6 +60,9 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('test_config', '')}",
                    description: 'Test configuration file',
                    name: 'test_config')
+            string(defaultValue: "${pipelineParams.get('base_versions', '')}",
+                   description: 'Base version in which the upgrade will start from',
+                   name: 'base_versions')
```

Comment:
To use this parameter you need to change usage of `pipelineParams.base_versions` to `params.base_versions` below. In other case it'll not use a value provided by an user on a build run.

Reply:
so maybe that's why it didn't work for me when i changed in the GUI
i will change it, and retry, thank you